src/fireblocks-sdk.ts

```diff
@@ -50,8 +50,8 @@ export class FireblocksSDK {
      * @param apiKey Your api key. This is a uuid you received from Fireblocks
      * @param apiBaseUrl The fireblocks server URL. Leave empty to use the default server
      */
-    constructor(privateKey: string, apiKey: string, apiBaseUrl: string = "https://api.fireblocks.io") {
-        this.authProvider = new ApiTokenProvider(privateKey, apiKey);
+    constructor(privateKey: string, apiKey: string, apiBaseUrl: string = "https://api.fireblocks.io", authProvider: IAuthProvider = undefined) {
+        this.authProvider = authProvider === undefined ? new ApiTokenProvider(privateKey, apiKey): authProvider;
```

Comment:
@kennethtxytqw Sorry for the late reply. Do you mind rewriting it as
`this.authProvider = authProvider ?? new ApiTokenProvider(privateKey, apiKey);`?
I'd be happy to merge

Reply:
Ah good catch!

Reply:
Fixed!